src/main/java/project/controller/DriverRiderController.java

```diff
@@ -41,4 +42,37 @@ public DriverRiderController(UserService userModel, MockDataService mockDataServ
         return new ResponseEntity<DriverRiderResponse>(driverRiderResponse, HttpStatus.OK);
     }
 
+    @RequestMapping(value = "/registerrider", method = RequestMethod.POST)
+    public ResponseEntity<Object> registerRriver(
+            @RequestParam(value="phone_number") int phoneNumber,
+            @RequestParam(value="low_price") int lowPrice,
+            @RequestParam(value="high_price") int highPrice,
+            @RequestParam(value="number_of_people") int numberOfPeople,
+            @RequestParam(value="location") String location,
+            @RequestParam(value="destination") String destination,
+            @RequestParam(value="message") String message,
+            @RequestParam(value="pickup_time_timestamp") long pickUpTimeTimestamp
```

Comment:
in the client/web you have to change the time that the user selected to timestamp,
if you don't know what timestamp is -> https://www.google.is/search?q=timestmap&oq=timestmap&aqs=chrome..69i57j69i60l3j69i59j69i61.1089j0j7&sourceid=chrome&es_sm=91&ie=UTF-8#q=what+is+timestamp

and you have to google how to change the object that you get from the user to timestamp before calling the endpoint